docs/core/install/linux-snap.md

```diff
@@ -130,6 +130,29 @@ The certificate location will vary by distro. Here are the locations for the dis
 | **OpenSUSE** | `/etc/ssl/ca-bundle.pem`                            |
 | **Solus**    | `/etc/ssl/certs/ca-certificates.crt`                |
 
+## Troubles resolving dotnet
```

Comment:
In case code-fencing is disallowed/discouraged in headings, it may be worth adding this to `no-loc` heading metadata.

Reply:
I checked some articles and it seems dotnet isn't being translated 👍